testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -2121,11 +2133,13 @@ def _check_universe(self, u):
         assert_equal(len(u.atoms[3].bonds), 2)
         assert_equal(len(u.atoms[4].bonds), 1)
         assert_equal(len(u.atoms[5].bonds), 1)
+        assert_('guess_bonds' in u._kwargs)
```

Comment:
Why does it matter if this kwarg is saved?  The previous test should have covered this